commands/build/lib/build.js

```diff
@@ -28,6 +31,20 @@ const resolveReplacementStrings = (replacementStrings) => {
   return replacementStrings;
 };
 
+const doINeedACarbonCopy = (argv) => {
+  const { carbon } = argv;
```

Comment:
Could we call it `reactNative` instead of `carbon`? That will make it a lot easier/useful if someone else want's to use it...

Reply:
Done!